pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -324,17 +327,20 @@ func (asw *actualStateOfWorld) SetVolumeMountedByNode(
 	if err != nil {
 		return fmt.Errorf("Failed to SetVolumeMountedByNode with error: %v", err)
 	}
+	// TODO: probably safer to switch getNodeAndVolume to return pointers?
```

Comment:
My thought was that getNodeAndVolume returns disconnected objects - so you always have to remember to do `volumeObj.nodesAttachedTo[nodeName] = nodeObj`.  I will just add a comment to getNodeAndVolume.